### PR TITLE
[JENKINS-33813] - fix issues when errors happen during setup wizard

### PR DESCRIPTION
**Pull request by kzantow**
This corrects a few things during the setup wizard:
- styling issues on the security token page after plugins installed
- styling on the error page
- add option to retry the setup wizard (e.g. if session times out)

image

This addresses: https://issues.jenkins-ci.org/browse/JENKINS-33813

@reviewbybees

### CODE REVIEW

File: core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly [hunk 1]
```diff
@@ -29,7 +29,6 @@
                     <label class="control-label" for="security-token">${%Administrator password}</label>
                     <input name="j_username" value="${j.setupWizard.initialSetupAdminUserName}" type="hidden"/>
                     <input id="security-token" class="form-control" type="password" name="j_password"/>
-                    <link rel="stylesheet" href="${j.installWizardPath}.css" type="text/css" />
```
Comment (kzantow) [inline core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly]: This was redundant, included by the html.jelly

**Comment by daniel-beck** [comment]
What is this about?

> styling issues on the security token page after plugins installed

**Comment by kzantow** [comment]
@daniel-beck if you started the setup, installed plugins it transitioned the state to something other than NEW. If you let the session time out and restarted, the security token page wasn't including the new CSS and looked awful.

File: core/src/main/resources/lib/layout/html.jelly [hunk 2]
```diff
@@ -155,9 +155,8 @@ ${h.initPageVariables(context)}
       </j:forEach>
     </j:if>
 
-  <j:invokeStatic var="j" className="jenkins.model.Jenkins" method="getActiveInstance" />
-    <j:set var="installState" value="${j.installState.name()}" />
-    <j:if test="${installState == 'NEW' || installState == 'INITIAL_PLUGINS_INSTALLING'}">
+    <j:invokeStatic var="j" className="jenkins.model.Jenkins" method="getInstance" />
+    <j:if test="${!j.installState.setupComplete}">
```
Comment (kzantow) [inline core/src/main/resources/lib/layout/html.jelly]: @daniel-beck this is the minor change - if session timed out and you had to enter the security token again, it was only including the correct CSS and such when install state was NEW and INITIAL_PLUGINS_INSTALLING, e.g. not configuring security or creating the admin user.

**Comment by jglick** [comment]
If you are sick of triggering new PR builds because of bogus-looking failures, review my #2188 and merge `master` into `2.0`…

**Comment by jglick** [comment]
:bee:

**Pull request merged**